stackslib/src/chainstate/nakamoto/miner.rs

```diff
@@ -536,7 +536,7 @@ impl NakamotoBlockBuilder {
             .mempool_settings
             .tenure_cost_limit_per_block_percentage
         {
-            if percentage < 100 {
+            if (1..100).contains(&percentage) {
```

Comment:
We could avoid this check altogether if a percentage of `100` in the config becomes a `None` instead of `Some(100)`.

Reply:
True, but i kept this additional check in case someone wrote code directly instead of using the raw config file. i.e. we shouldn't attempt to INCREASE the amount of block space no matter what. for example, an integration test could easily by pass this config check. Should I do an expect on anything earlier on that exceeds this range?